src/commands/aem/rde/install.js

```diff
@@ -271,6 +294,33 @@ class DeployCommand extends BaseCommand {
 }
 
 /**
+ *
+ * @param inputPath
+ */
+function guessPath(inputPath) {
+  const JCR_ROOT_SUBPATH = '/' + JCR_ROOT + '/';
+  if (!inputPath.includes(JCR_ROOT)) {
+    return;
+  }
+  const extension = inputPath.substring(inputPath.lastIndexOf('.'));
+  if (inputPath.endsWith('.content.xml')) {
+    return inputPath.substring(
+      inputPath.lastIndexOf(JCR_ROOT_SUBPATH) + JCR_ROOT_SUBPATH.length,
+      inputPath.lastIndexOf('/')
+    );
+  } else if (extension === '.xml') {
+    return inputPath.substring(
+      inputPath.lastIndexOf(JCR_ROOT_SUBPATH) + JCR_ROOT_SUBPATH.length,
+      inputPath.lastIndexOf('.')
+    );
+  }
+  return inputPath.substring(
+    inputPath.lastIndexOf(JCR_ROOT_SUBPATH) + JCR_ROOT_SUBPATH.length
+  );
+}
```

Comment:
I'm suggesting the following simplification to (a) make it clearer which end-markers we are looking for  ("/.content.xml" or ".xml"), and (b) reduce duplicate code.

```suggestion
function guessPath(inputPath) {
  const JCR_ROOT_SUBPATH = '/' + JCR_ROOT + '/';
  if (!inputPath.includes(JCR_ROOT)) {
    return;
  }
  const extension = inputPath.substring(inputPath.lastIndexOf('.'));
  if (extension === '.xml') {
    return inputPath.substring(
      inputPath.lastIndexOf(JCR_ROOT_SUBPATH) + JCR_ROOT_SUBPATH.length,
      inputPath.lastIndexOf(inputPath.endsWith('/.content.xml') ? '/.content.xml' : '.xml')
    );
  }
  return inputPath.substring(
    inputPath.lastIndexOf(JCR_ROOT_SUBPATH) + JCR_ROOT_SUBPATH.length
  );
}
```

Reply:
I will add this into the tets coverage as I have slightly changed this already, if that's okay with you?

Reply:
Ok, works for me!